etc/errors/anvil.json

```diff
@@ -19,7 +19,215 @@
           "component_name": "AnvilEnvironment",
           "component_code": 1,
           "identifier_encoding": "env",
-          "errors" : []
+          "errors" : [
+            {
+              "name": "InvalidArguments",
+              "code": 1,
+              "message": "Invalid arguments: {details}.",
+              "fields": [
+                {
+                  "name": "details",
+                  "type": "string"
+                },
+                {
+                  "name": "arguments",
+                  "type": "string"
+                }
+              ],
+              "doc": {
+                "summary": "Invalid command line arguments provided.",
+                "description": "There are missing or invalid command line arguments, or an invalid combination of arguments is provided.",
+                "likely_causes": [
+                  {
+                    "cause": "Missing an essential argument or an incorrect value is provided.",
+                    "fixes": [
+                      "Check the short help on the CLI arguments using `anvil-zksync -h`"
+                    ]
+                  }
+                ]
+              }
+            },
+            {
+              "name": "ServerStartupFailed",
+              "code": 2,
+              "message": "Failed to start server at {host_requested}:{port_requested}, and failed to fall back to {host_requested}:{port_fallback_retried}: {details}.",
```

Comment:
Isn't there supposed to be a wrapped IO error here with what went wrong exactly?

Reply:
I have intended the field "details" to hold it, wdyt?